packages/bitcore-node/src/routes/api/address.ts

```diff
@@ -30,6 +31,25 @@ router.get('/:address', function(req, res) {
   ChainStateProvider.streamAddressUtxos(payload);
 });
 
+router.get('/:address/coins', async function (req, res) {
+  let { address, chain, network } = req.params;
+
+  try {
+    let coins = await CoinStorage.collection.find({ address, chain, network }).toArray(); //rename coins
```

Comment:
Since there's no limit on this, I think this is a DoS vector.

If you want something that does this, we'll need to use `Storage.apiStream()` with limits and paging.

You'd need to Transform the original stream and push {inputs, outputs} for each coin